### DISCUSSION TITLE
How to start with Shiny.NET example in Sample.Maui folder in VS2022

### DISCUSSION THREAD

**Discussion by ChumKiu** [shinyorg/shiny#1493] [Q&A]
Dear all,
I'm try to run some sample project from the unzipped archive https://github.com/shinyorg/shiny.
I've loaded inside VS2022 the file "Sample.Maui.csproj".
I've set the Sample.Maui as starting project.
Running compilation give many errors and I do not know if my flow is the right one to execute some sample or if I've to do in different way.
Appreciated some hint/help to start.
Thanks and best regards.
Chum

**Comment by aritchie** [comment]
You have to give more details.  I can't even make a guess.  What are the errors you're getting?  Do you have the .NET MAUI workloads installed?  If so, what version?  What branch are you trying to compile?
